napari/utils/tree/group.py

```diff
@@ -30,14 +31,24 @@ class Group(NestableEventedList[NodeType], Node):
         A name/id for this group, by default "Group"
     """
 
-    def __init__(self, children: Iterable[NodeType] = (), name: str = "Group"):
-        NestableEventedList.__init__(self, children, basetype=Node)
+    def __init__(
+        self,
+        children: Iterable[NodeType] = (),
+        name: str = "Group",
+        basetype=Node,
+    ):
         Node.__init__(self, name=name)
+        NestableEventedList.__init__(self, data=children, basetype=basetype)
+        self._selection: Selection[NodeType] = Selection()
+
+    @property
+    def selection(self) -> Selection[NodeType]:
```

Comment:
Just wondering how we'd think about serializing this now as it is both a List and has an attribute, it could just be that we can never serialize the selection state, and we just ignore it.

Reply:
see `Select._encode_json` for serialization.  It becomes a dict with a list and a scalar.